src/popup/views/SignTransaction/index.tsx

```diff
@@ -149,8 +149,6 @@ export const SignTransaction = () => {
         }: TransactionInfoResponse,
         i: number,
       ) => {
-        const formattedAmount = new BigNumber(amount).toFormat(2);
-        const formattedBuyAmount = new BigNumber(buyAmount).toFormat(2);
         const operationIndex = i + 1;
```

Comment:
on a non payment transaction (set options, for ex) we were hitting errors bc amount/buyAmount were undefined